plugins/examples/custom-significance-heuristic/build.gradle

```diff
@@ -5,7 +5,7 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-apply plugin: 'elasticsearch.internal-es-plugin'
```

Comment:
we reference the notice and license file via rootProject in those example project files. I think that doesn't work anymore with this setup as rootProject is not ES root

Reply:
Ah, good catch. I've copied the NOTICE and LICENSE files into the example plugins project.